INSTALL.md

```diff
@@ -68,6 +68,25 @@ source, building and installing them. They can be located here:
 
 To get md2man-roff, see there page at: <https://github.com/sunaku/md2man>
 
```

Comment:
@kaiserkyu I would change your commit subject line and message to say that you are adding a `build from source` example rather than something that's Fedora specific.